R/apa_print_manova.R

```diff
@@ -0,0 +1,115 @@
+
+#' @param test For MANOVA, the multivariate test statistic to be reported, see \code{\link{summary.manova}}.
+#' @rdname apa_print.aov
+#' @export
+
+apa_print.manova <- function(x, test = "Pillai", ...) {
+
+  tidied_x <- as.data.frame(
+    x = broom::tidy(x, test = test, ...)
+    , stringsAsFactors = FALSE
+  )
+  print_manova(x = tidied_x, ...)
+}
+
+
+#' @rdname apa_print.aov
+#' @export
+
+apa_print.summary.manova <- function(x, ...) {
+
+  tidied_x <- as.data.frame(
+    x = broom::tidy(x, ...)
+    , stringsAsFactors = FALSE
+  )
+  print_manova(x = tidied_x, ...)
+}
+
+
+
+#' @keywords internal
+
+print_manova <- function(
+  x
+  , in_paren = FALSE
+) {
+
+
+
+  estimate_names <- c(
+    pillai = "V"
+    , wilks = "\\Lambda"
+    , hl = "T"
+    , roy = "\\theta"
+  )
+
+
+  estimate_names_with_dollars <- paste0("$", estimate_names, "$")
+  names(estimate_names_with_dollars) <- names(estimate_names)
+
+  multivariate_stat <- intersect(names(estimate_names), colnames(x))
+
+  if(length(multivariate_stat) == 0L) {
+    stop("No multivariate test statistic found in results object.")
+  } # move this to input validation
+
+  col_names <- c(
```

Comment:
Hmm, I'm not sure about this bit. I can see you followed the behavior of `print_anova()` here. However, I wonder whether it would make more sense to keep the naming of the columns consistent with the `broom` output. I think this is what we do for regression, too. It would probably make sense to tackle this in a separate PR, but I'd be interested in your opinion on this.

Reply:
You mean term, statistic, and num.df, den.df instead of Effect, F, df1, and df2?

1. I think it would be very confusing to coin a column "term" and then label it "Effect".
2. I think the output from broom here is quiet inconsistent: coining one column "pillai" and the other "statistic" is less consistent than using "pillai" and "F" (it would also be  to use "multivariate.statistic" and "univariate.statistic")

Reply:
Given that you implemented the MANOVA methods in `broom`, we know who's to blame for the inconsistency here. :stuck_out_tongue_winking_eye:

I understand you concern that indexing with `$term` although the column is printed with the heading `Effect` may be confusing. I suggest we think this through a little more. I think it's definitely desirable to be as consistent as possible in our naming of `apa_result_table`-objects.